app/controllers/forest_liana/application_controller.rb

```diff
@@ -25,14 +25,10 @@ def self.papertrail?
       # NOTICE: The Forest user email is returned to track changes made using
       #         Forest with Papertrail.
       define_method :user_for_paper_trail do
-        forest_user['data']['data']['email']
+        @jwt_decoded_token['email']
       end
     end
 
-    def forest_user
-      @jwt_decoded_token
-    end
```

Comment:
[Regression] `forest_user` is a function used for customization, see this example (in Rails):
https://docs.forestadmin.com/documentation/reference-guide/routes#what-is-a-smart-route

So I am not sure we should delete this. We should add a comment to explain what is this method for by the way.